docs/devcontainers.md

```diff
@@ -0,0 +1,164 @@
+# Devcontainer support
+
+A [development container](https://containers.dev) (or devcontainer for short) allows you to use a container as a development environment. It can be used to develop and run applications in the isolated environment that is persistent across different developers. This page describes the configuration for VS Code. You can refer to [this](https://containers.dev/supporting) page for other supported tools.
+
+## Requirements
+
+- Install [VS Code](https://code.visualstudio.com/docs/setup/setup-overview)
+- Install [Devcontainers Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+- Navigate to the root of Artery repository
```

Comment:
"Navigate" is a little bit imprecise. I suppose you mean "Open the root directory of your cloned Artery repository in VSCode".